lib/arInstall.class.php

```diff
@@ -298,7 +298,8 @@ public static function checkSearchConnection($options)
         }
 
         $curlHttpCode = curl_getinfo($curl, CURLINFO_HTTP_CODE);
-        curl_close($curl);
+        // set curl handle to null to close connection
+        $curl = null;
```

Comment:
I am afraid that setting a variable to null is not the same as closing... This is a code-smell for a later problem of memory leaks and unused resources.